ci-operator/config/ViaQ/logging-fluentd/ViaQ-logging-fluentd-v1.7.4.yaml

```diff
@@ -52,7 +52,7 @@ base_images:
     namespace: logging
     tag: logging-elasticsearch6
   logging-eventrouter:
-    name: 5.y
+    name: 6.y
```

Comment:
We should consider reviewing the fluent jobs to see what versions we actually need.  I think there are several of these which can be removed

Reply:
I've only kept the two branches listed in the version matrix for our current versions.